### DISCUSSION TITLE
Any guide for Google Colab Deep fake video +18 content?

### DISCUSSION THREAD

**Discussion by Boldomoli** [FurkanGozukara/Stable-Diffusion#34] [Q&A]
I have below problem

![image](https://github.com/FurkanGozukara/Stable-Diffusion/assets/101622763/1271ab5f-4a85-477d-939f-d540e0dddb92)

**Comment by FurkanGozukara** [comment]
nope we don't support +18

**Comment by Boldomoli** [comment]
Sad
